src/test/shared/utilities/vscodeUtils.test.ts

```diff
@@ -99,3 +102,60 @@ describe('buildMissingExtensionMessage()', function () {
         )
     })
 })
+
+describe('isExtensionFirstUse', function () {
```

Comment:
Could probably avoid this test code if we had a globalState module

Reply:
This test was simplified, so not as bad as before. There was a small intricacy managing some state for the current extension session (until extension is stopped) which required this more complex solution.